.github/actions/display-jest-test-summary/README.md

```diff
@@ -0,0 +1,28 @@
+# Display Jest Test Summary
+Action to output results of jest tests to Github summary
+
+####  Prerequisites
+Jest tests ran and HTML file generated via [jest-html-reporter](https://www.npmjs.com/package/jest-html-reporter)
+
+[Cheerio installed](https://cheerio.js.org/)
+
+[fs-extra installed](https://www.npmjs.com/package/fs-extra)
+
+####  Context
+This action allows the workflow to add a summary of jest test passes and failures to the GitHub summary
+This does not display details of test failures, it can be used in addition to the 'github-actions' reporter as part of 
+the jest config
+
+####  Inputs
+test_report_path (required): Path of test report HTML file
+
+####  Outputs
+Table of jest test results displayed on Github summary
+
+####  Usage
+```yaml
+- name: Display Jest Test Summary
+  uses: dvsa/.github/.github/actions/display-jest-test-summary@main
+  with:
+    test_report_path: 'test-report.html'
```

Comment:
Where does this file come from? Do you need to save/upload the test report from the test run?

Reply:
@alexisc-kainos  please answer above

Reply:
@AnnaDodson It's based on the jest config. MOT Recalls uses jest-html-reporter which creates a file test-report.html when tests are run. It can be set the path to which your application saves it's jest report